### PR TITLE
Don't import db1

### PR DESCRIPTION
**Pull request by Powersource**
Fixes https://github.com/ssbc/ssb-tribes/issues/98

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
   "devDependencies": {
     "cross-env": "^7.0.3",
     "is-canonical-base64": "^1.1.1",
-    "scuttle-testbot": "^2.1.0",
+    "scuttle-testbot": "github:ssbc/scuttle-testbot#no-use",
```
Comment (Powersource) [inline package.json:45]: https://github.com/ssbc/scuttle-testbot/pull/26

**Comment by Powersource** [comment]
Hmm replicate is failing now

```
# replicate group members
/home/me/prj/ssb/ssb-tribes/node_modules/ssb-replicate/legacy.js:72
  ssbServer.post(function (msg) {
            ^

TypeError: ssbServer.post is not a function
    at Object.module.exports (/home/me/prj/ssb/ssb-tribes/node_modules/ssb-replicate/legacy.js:72:13)
    at Object.init (/home/me/prj/ssb/ssb-tribes/node_modules/ssb-replicate/index.js:14:30)
    at /home/me/prj/ssb/ssb-tribes/node_modules/secret-stack/lib/api.js:28:34
    at Array.forEach (<anonymous>)
    at create (/home/me/prj/ssb/ssb-tribes/node_modules/secret-stack/lib/api.js:27:24)
    at createTestBot (/home/me/prj/ssb/ssb-tribes/node_modules/scuttle-testbot/index.js:53:16)
    at TestBot (/home/me/prj/ssb/ssb-tribes/test/helpers/test-bot.js:32:15)
    at Test.<anonymous> (/home/me/prj/ssb/ssb-tribes/test/replicate.test.js:26:15)
    at Test.run (/home/me/prj/ssb/ssb-tribes/node_modules/tape/lib/test.js:113:28)
    at Immediate.next (/home/me/prj/ssb/ssb-tribes/node_modules/tape/lib/results.js:157:7)
```

doing `.use(require('ssb-db2/compat/post'))` didn't seem to even change the error (that compat plugin seems to add `ssb.db.post`, not `ssb.post`?) and even if that helped there's a few other functions used in replicate that we'd also need compat for (there are for at least some of them though?)

not sure exactly what to do now, but it's also the middle of the night

**Comment by Powersource** [comment]
fixed the post compat here https://github.com/ssbc/ssb-db2/pull/426

File: package.json [hunk 2]
```diff
@@ -42,9 +42,9 @@
...
     "ssb-box2": "^7.2.0",
-    "ssb-db2": "^7.1.1",
+    "ssb-db2": "github:ssbc/ssb-db2#fix-post-compat",
```
Comment (Powersource) [inline package.json:47]: https://github.com/ssbc/ssb-db2/pull/426

File: test/rebuild.test.js [hunk 3]
```diff
@@ -259,12 +251,12 @@ test('rebuild from listen.addMember', t => {
...
 
       if (heardCount === 2) {
-        checkRebuildDone(() => {
+        setTimeout(() => {
           A.close(err => {
             t.error(err, 'A closed')
             t.end()
           })
-        })
+        }, 500)
```
Comment (mixmix) [inline test/rebuild.test.js:259]: is that works, that's cool. I prefer more deterministic things, but all good

**Review by mixmix — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Powersource) [reply package.json:45]: v2.2.0

File: package.json (hunk 2 above)
Comment (Powersource) [reply package.json:47]: v8.0.0

**Pull request merged**